packages/cloud-cognitive/src/components/AddSelect/AddSelectRow.js

```diff
@@ -0,0 +1,227 @@
+//
+// Copyright IBM Corp. 2022, 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+import React, { useRef, useEffect, useCallback } from 'react';
+import { Button, Dropdown } from '@carbon/react';
+import { ChevronRight, View } from '@carbon/react/icons';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { pkg } from '../../settings';
+import { AddSelectFormControl } from './AddSelectFormControl';
+
+const blockClass = `${pkg.prefix}--add-select__selections`;
+const componentName = 'AddSelectList';
+
+export let AddSelectRow = ({
+  appliedModifiers,
+  displayMetalPanel,
+  index,
+  focus,
+  item,
+  metaIconDescription,
+  modifiers,
+  multi,
+  multiSelection,
+  navIconDescription,
+  parentId,
+  setAppliedModifiers,
+  setDisplayMetaPanel,
+  setFocus,
+  setMultiSelection,
+  setParentSelected,
+  setSingleSelection,
+  singleSelection,
+}) => {
+  const ref = useRef(null);
+
+  useEffect(() => {
+    if (focus === index) {
+      ref.current.focus();
+    }
+  }, [focus, index]);
+
+  const isSelected = () => {
+    if (multi) {
+      return multiSelection.includes(item.id);
+    }
+    return item.id === singleSelection;
+  };
+
+  const getTabIndex = () => {
+    // on initial load make the first item tabbable
+    if (index === 0 && focus === '') {
+      return 0;
+    }
+
+    // make it so only the last focused item is tabbable
+    if (focus === index && focus !== '') {
+      return 0;
+    }
+
+    // make unfocused items un-tabbable
+    return -1;
+  };
+
+  const focusHandler = useCallback(
+    (reset) => {
+      setFocus(reset ? '' : index);
+    },
+    [setFocus, index]
+  );
+
+  const handleSingleSelection = () => {
+    setSingleSelection(item.id);
+  };
+
+  const onSelectKeyDown = ({ key }) => {
+    if (key === 'Enter') {
```

Comment:
Hopefully a quick fix, but can we get this working with both <kbd>Enter</kbd> and <kbd>Space</kbd>? I noticed that the actual checkboxes in MultiAddSelect require `space` so this inconsistency feels unexpected.